pos_order_remove_line/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+**English**
```

Comment:
[nitpicking] I think that it is unncessary, as all the description are allways in English.

Reply:
Yep! Done!